src/clusterfuzz/_internal/system/archive.py

```diff
@@ -66,53 +75,61 @@ def _is_attempting_path_traversal(archive_name, output_dir, filename) -> bool:
 
 @dataclasses.dataclass
 class ArchiveMemberInfo:
-  """Represents an archive member.
+  """Represents an archive member. A member can either be a file or a directory.
+  Members:
+    name: the name of the archive member.
```

Comment:
Can this be a relative path for directory entries? e.g. `foo/bar`?

Reply:
Yes!

Reply:
Thanks! Let's document that then.